source/marsGameSound.js

```diff
@@ -20,15 +20,15 @@ this.initialize = function() {
     for ( var i = 0; i < soundSet.length; ++i ) {
         this.loadSound( soundSet[i] );
     }
-    this.future( 0 ).setUpSubtitles();
+    this.future( 0.01 ).setUpSubtitles();
```

Comment:
There was a conversation I had with... somebody.  @BrettASwift?  @davideaster?  Anyway, the sense I came away with was that a future( 0 ) wouldn't always get you past all the events you needed to wait for - you wanted to actually put it at a time in the future, so that all the things scheduled for the current time would truly be guaranteed to be done.